CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 # NEXT RELEASE
 
 ### Enhancements
-* <New feature description> (PR [#????](https://github.com/realm/realm-core/pull/????))
-* None.
+* Implement initial subscription callback for synchronized realms. ([#5962](https://github.com/realm/realm-core/issues/5962))
```

Comment:
I find this a bit vague. You can consider rephrasing into something like: `Added support for configuring initial subscriptions for synchronized realms`.